samples/matter/lock/README.rst

```diff
@@ -536,6 +641,111 @@ After building the sample and programming it to your development kit, complete t
+     The maximum value is defined by the :kconfig:option:`CONFIG_LOCK_MAX_WEEKDAY_SCHEDULES_PER_USER` kconfig option.
+   * ``UserIndex`` is the user index defined in the previous step, starting from ``1``.
+   * ``DaysMask`` is a bitmap of numbers of the week days starting from ``0`` as a Sunday and finishing at ``6`` as a Saturday.
+     For example, to assign this schedule to Tuesday, Thursday and Saturday you need to provide ``84`` because it is equivalent to the ``01010100`` bitmap.
+   * ``StartHour`` is the starting hour for the week day schedule.
+   * ``StartMinute`` is the starting minute for the week day schedule.
+   * ``EndHour`` is the ending hour for the week day schedule.
+   * ``EndMinute`` is the ending hour for the week day schedule.
+   * ``node-id`` is the device node ID.
+   * ``endpoint-id`` is the Matter door lock endpoint, in this sample assigned to ``1``.
+
+   For example, use the following command to set a ``week-day`` schedule for Tuesday, Thursday and Saturday to start at 7:30 AM and finish at 10:30 AM dedicated for user with ID ``1``:
+
+   .. code-block:: console
+
+      ./chip-tool doorlock set-week-day-schedule 1 1 84 7 30 10 30 1 1
+
+
+#. Set the example ``year-day`` schedule using the following command:
+
+   ./chip-tool doorlock set-year-day-schedule YearDayIndex UserIndex LocalStartTime LocalEndTime node-id endpoint-id
+
+   Where:
+
+   * ``YearDayIndex`` is an index of the new schedule, starting from ``1``.
+     The maximum value is defined by the :kconfig:option:`CONFIG_LOCK_MAX_YEARDAY_SCHEDULES_PER_USER` kconfig option.
+   * ``UserIndex`` is the user index defined in the previous step, starting from ``1``.
+   * ``LocalStartTime`` is the starting time in Epoch Time.
+   * ``LocalEndTime`` is the ending time in Epoch Time.
+   * ``node-id`` is the device node ID.
+   * ``endpoint-id`` is the Matter door lock endpoint, in this sample assigned to ``1``.
+
+   Both ``LocalStartTime`` and ``LocalEndTime`` are in seconds with the local time offset based on the local timezone and DST offset on the day represented by the value.
+
+   For example:
+
+   .. code-block:: console
+
+      ./chip-tool doorlock set-week-day-schedule 1 1 60 120 1 1
+
+#. Set the example ``holiday`` schedule using the following command:
+
+   ./chip-tool doorlock set-holiday-schedule HolidayIndex LocalStartTime LocalEndTime OperatingMode destination-id endpoint-id
+
+   Where:
+
+   * ``HolidayIndex`` is an index of the new schedule, starting from ``1``.
+   * ``LocalStartTime`` is the starting time in Epoch Time.
+   * ``LocalEndTime`` is the ending time in Epoch Time.
+   * ``OperatingMode`` is the operating mode described in the :ref:`matter_lock_scheduled_timed_access` section of this guide.
+   * ``node-id`` is the device node ID.
+   * ``endpoint-id`` is the Matter door lock endpoint, in this sample assigned to ``1``.
+
+   For example, to setup a ``holiday-schedule`` with type ``Vacation`` use the following command:
```

Comment:
Expand this to explain all the values used as well.

Reply:
Done